core/jvm/src/main/scala/plotly/JavaTime.scala

```diff
@@ -0,0 +1,30 @@
+package plotly
+
+import java.time._
+import plotly.element.{LocalDateTime => PlotlyLocalDateTime}
+
+object JavaTime {
+
+  implicit def fromJavaLocalDateTime(javaLocalDateTime: LocalDateTime): PlotlyLocalDateTime =
+    PlotlyLocalDateTime(
+      javaLocalDateTime.getYear,
+      javaLocalDateTime.getMonthValue,
+      javaLocalDateTime.getDayOfMonth,
+      javaLocalDateTime.getHour,
+      javaLocalDateTime.getMinute,
+      javaLocalDateTime.getSecond,
+    )
+
+  implicit def fromJavaInstant(javaInstant: Instant): PlotlyLocalDateTime =
+    fromJavaOffsetDateTime(javaInstant.atOffset(ZoneOffset.UTC))
+
+  implicit def fromJavaOffsetDateTime(javaOffsetDateTime: OffsetDateTime): PlotlyLocalDateTime =
+    fromJavaLocalDateTime(javaOffsetDateTime.toLocalDateTime)
+
+  implicit def fromJavaZonedDateTime(javaZonedDateTime: ZonedDateTime): PlotlyLocalDateTime =
+    fromJavaLocalDateTime(javaZonedDateTime.toLocalDateTime)
+
```

Comment:
Worth noting that there are some potential gotchas here around converting zoned datetimes caused by dropping the zone component. For example, converting a list of datetimes with differing zones/offsets may lead to unexpected results. I'm happy to add a Scaladoc warning if needed.

Reply:
Feel free to mention it in a scaladoc comment if you wish, yes.